### DISCUSSION TITLE
原生js，非vue等，怎么注入组件，拖拽创建节点啊

### DISCUSSION THREAD

**Discussion by minliuhua** [didi/LogicFlow#1849] [Q&A] [answered]
[logic-flow.zip](https://github.com/user-attachments/files/16887277/logic-flow.zip)
这种方式不行，用<script>引入也不行
![image](https://github.com/user-attachments/assets/fe552411-9716-4d97-92b3-9c5b1b375e2a)

**Comment by FanSun521** [comment, marked as answer]
[logic-flow.zip](https://github.com/user-attachments/files/16902346/logic-flow.zip)


去查查浏览器怎么使用cdn + 模块化就行

**Comment by minliuhua** [comment]
好的
